test/attitude.cpp

```diff
@@ -130,18 +147,18 @@ int main()
     double rad2deg = 180.0 / M_PI;
 
     // euler dcm round trip check
-    for (int roll = -90; roll <= 90; roll += 90) {
-        for (int pitch = -90; pitch <= 90; pitch += 90) {
-            for (int yaw = -179; yaw <= 180; yaw += 90) {
+    for (double roll = -90; roll <= 90; roll += 90) {
+        for (double pitch = -90; pitch <= 90; pitch += 90) {
+            for (double yaw = -179; yaw <= 180; yaw += 90) {
                 // note if theta = pi/2, then roll is set to zero
```

Comment:
using doubles here fixed a compile if you try to compile the tests without coverage flags on, not sure if we should add that to travis or not, not really a big deal